lcserve/__main__.py

```diff
@@ -1029,6 +1049,58 @@ async def remove(app_id):
     await remove_app_on_jcloud(app_id)
 
 
+@click.group(help="Job related operations.")
+def job():
+    pass
+
+
+@job.command('list', help='List all jobs for an app.')
+@click.argument('flow-id')
+@click.help_option('-h', '--help')
+@syncify
+async def list_jobs(flow_id):
+    await list_jobs_on_jcloud(flow_id)
+
+
+@job.command('get', help='Get job details.')
+@click.argument('job-name')
+@click.argument('flow-id')
+@click.help_option('-h', '--help')
+@syncify
+async def get_job(job_name, flow_id):
+    await get_job_on_jcloud(job_name, flow_id)
+
+
+@job.command('create', help='Create a job.')
+@click.argument('flow-id')
+@click.argument('job-name')
+@click.option('--param', multiple=True, help='Parameter in the form key=value.')
+def create_job(flow_id, job_name, param):
+    from rich import print
+
+    token = Auth.get_auth_token()
+    if not token:
+        print('You are not logged in, please login using [b]jcloud login[/b] first.')
+
+    params = dict(x.split('=') for x in param)
```

Comment:
What does this do? Can you please add a description?

Reply:
removed, I changed to this kind of input:

```
python -m lcserve job create langchain-21a817c931 myjob --params doc_name state_of_the_union.txt --params question 'What did the president say about Ketanji Brown Jackson'
```